ModelViewer/ModelViewer/ViewerGraphicsWindow.h

```diff
@@ -13,9 +13,28 @@ class ViewerGraphicsWindow : public OpenGLWindow
 
     void initialize() override;
     void render() override;
+    void resetView();
 
     bool loadModel(QString filepath = QString());
 
+    // Mouse variables
+    QMatrix4x4 sceneMatrix;
+    float viewportX = 0;
+    float viewportY = 0;
+
+    // Mouse state
+    bool leftMousePressed = false;
+    bool rightMousePressed = false;
+
+    // Mouse settings | % adjustment
+    float viewportXSensitivity = 1;
```

Comment:
Do you want to mark these all as const?

`const float .....`

Reply:
The user might want to change those values down the line.

I'll add an issue regarding a settings menu.